js/imageeditor/shared/ImageEditor.svelte

```diff
@@ -605,6 +605,7 @@
 				on:set_zoom={(e) => handle_zoom_change(e.detail)}
 				on:zoom_in={() => zoom_in_out("in")}
 				on:zoom_out={() => zoom_in_out("out")}
+				on:clear={() => dispatch("clear")}
```

Comment:
I think we can just dispatch clear in the `remove_image` event handler without adding the other clear event to the Controls component.